src/main/java/uk/gov/hmcts/ccd/domain/model/common/CatalogueResponseGroup.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.ccd.domain.model.common;
+
+public enum CatalogueResponseGroup {
+
+    SUCCESS("00"),
+    VALIDATION("01"),
+    CALLBACK("02");
+
+    final String code;
+
+    CatalogueResponseGroup(final String code) {
```

Comment:
code as input to constructor and code returned from getCode() are inconsistent, reducing readability. Let's rename the parameter like int groupId. We will also need a groupId attribute and it's getter.